modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -91,6 +93,22 @@ def veteran_from_headers(with_gender: false)
         vet
       end
 
+      def validate_ccg_token!
+        client = TokenValidation::V2::Client.new(api_key: Settings.claims_api.token_validation.api_key)
+        root_url = request.base_url == 'http://localhost:3000' ? 'https://sandbox-api.va.gov' : request.base_url
```

Comment:
this seems clean enough to me, but let me know if there's a better way to accomplish this